.github/workflows/zscaler-iac-scan.yml

```
@@ -36,21 +36,22 @@ jobs:
       - name : Code Checkout
         uses: actions/checkout@v4
       - name : Zscaler IAC Scan
+        if: secrets.ZSCANNER_CLIENT_ID != '' && secrets.ZSCANNER_CLIENT_SECRET != ''
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)